packages/jest-cli/src/cli/args.js

```diff
@@ -493,6 +498,12 @@ export const options = {
     description: 'Alias for --env',
     type: 'string',
   },
+  testEnvironmentOptions: {
+    description:
+      'Test environment options that will be passed to the testEnvironment. ' +
+      'The relevant options depend on the environment',
+    type: 'Object',
```

Comment:
this should be a `'string'`

Reply:
https://facebook.github.io/jest/docs/en/configuration.html#testenvironmentoptions-object this says Object ??

Reply:
`yargs` (we use it to parse CLI flags) doesn't recognize this type. It's more like stringified object, but still a string in yargs-land. Please add somewhere in the description that this is an object.

Reply:
I can put similar like testFailureExitCode which has type: 'string' and comment next to it says number